README.md

```diff
@@ -1,406 +1,75 @@
 
-### AccountExtensions
+## Release notes, roadmap and SLA
 
-Finally, you can create a `ClaimsPrincipal` from an instance of MSAL.NET `IAccount`, using the `ToClaimsPrincipal` method in `AccountExtensions`.
+### Release notes and roadmap
 
-<img alt="AccountExtensions" src="https://user-images.githubusercontent.com/13203188/62538259-341b5880-b807-11e9-9328-a094f79a0874.png" width="60%"/>
+The Microsoft Identity Web roadmap is available from [Roadmap](https://github.com/AzureAD/microsoft-identity-web/wiki/#roadmap) in the [Wiki pages](https://github.com/AzureAD/microsoft-identity-web/wiki), along with release notes.
 
-### Troubleshooting your web app or web API
+### Support SLA
 
-To troubleshoot your web app, you can set the `subscribeToOpenIdConnectMiddlewareDiagnosticsEvents` optional boolean to `true` when you call `AddSignIn`. This displays in the output window the progression of the OpenID connect message through the OpenID Connect middleware (from the reception of the message from Azure Active directory to the availability of the user identity in `HttpContext.User`).
+Microsoft Identity Web is in preview.
 
-<img alt="OpenIdConnectMiddlewareDiagnostics" src="https://user-images.githubusercontent.com/13203188/62538366-75ac0380-b807-11e9-9ce0-d0eec9381b78.png" width="75%"/>
+- Major versions are supported for twelve months after the release of the next major version.
+- Minor versions older than N-1 are not supported.
+  > Minor versions are bugfixes or features with non-breaking (additive) API changes.  It is expected apps can upgrade.  Therefore, we will not patch old minor versions of the library. You should also confirm, in issue repros, that you are using the latest minor version before the Microsoft Identity Web team spends time investigating an issue.
+- Microsoft Identity Web is expected to become Generally Available in Fall 2020.
 
-To troubleshoot your web API, you can set the `subscribeToJwtBearerMiddlewareDiagnosticsEvents` optional boolean to `true` when you call `AddProtectedWebApi`. Enabling these diagnostics displays in the output window the progression of the OAuth 2.0 message through the JWTBearer middleware (from the reception of the message from Azure Active directory to the availability of the user identity in `HttpContext.User`).
+## Using Microsoft Identity Web
 
-<img alt="JwtBearerMiddlewareDiagnostics" src="https://user-images.githubusercontent.com/13203188/62538382-7d6ba800-b807-11e9-9540-560e7129197b.png" width="65%"/>
+- The conceptual documentation is currently available from the [Wiki pages](https://github.com/AzureAD/microsoft-identity-web/wiki).
+- Code samples are available:
+  > [Web app samples](https://github.com/AzureAD/microsoft-identity-web/wiki/web-app-samples)
+  > [Web API samples](https://github.com/AzureAD/microsoft-identity-web/wiki#web-api-samples)

```

Comment:
Put these code links on the separate lines or just put in one line:

Code samples are available for [Web Apps](https://github.com/AzureAD/microsoft-identity-web/wiki/web-app-samples) and [Web APIs](https://github.com/AzureAD/microsoft-identity-web/wiki#web-api-samples).